mod/dfrn_request.php

```diff
@@ -514,6 +514,11 @@ function dfrn_request_post(App $a) {
 					return; // NOTREACHED
 				}
 
+				if (blocked_url($url)) {
+					notice( t('Blocked domain') . EOL);
```

Comment:
Here as well. I think I would like to know why the contact request was refused, in more detail then just the general hint.